### ISSUE TITLE
Button: Light Green Button Color Contrast

### ISSUE THREAD

**Issue by bpmcneilly** [benetech/MathShare#953]
A shade of light green (#48cc70) is used in a series of buttons within the product. These include the add problem button (`class=“addStep”`) and the Finish button (`class=“finish”`). In addition to the button component, this color is used in the Symbols button when it is active (`class=“symbolsActive”`).

Each of these buttons contain white (#ffffff) text. These two colors form a contrast ratio of 2.07:1, far below the 4.5:1 required for normal sized text. 

#### Suggested Solution
While the green could be darkened to achieve an appropriate color contrast ratio, a significantly darker green hue will be required to create a 4.5:1 color contrast ratio with white text. In this case, it may be easier to change the text to black (#000000).

**Comment by johnhbenetech** [comment]
Hey @alexrcabral @benetechMatt please suggest an alternate color scheme for these buttons that meets the contrast required.

If I don't receive a suggestion by Monday, I will make a decision and move forward

**Comment by johnhbenetech** [comment]
@rupeshparab Let's go with #206F39 button color until we here otherwise and we can change it
